coreLib/src/main/java/dev/kosmx/playerAnim/api/first_person/FirstPersonRenderState.java

```diff
@@ -0,0 +1,21 @@
+package dev.kosmx.playerAnim.api.first_person;
+
+
+public class FirstPersonRenderState {
```

Comment:
This is important to stay in the API package.
Checking for ongoing render cycle state can be important for some features.